docs/.vitepress/config.js

```diff
@@ -28,11 +28,11 @@ const pdColorClasses = ['blue', 'cyan', 'fuchsia', 'indigo', 'pink', 'purple', '
 // if you add a class somewhere in code, and it doesn't work, add that class here (no time to explain)
 const docsClasses = [
   'aspect-square', 'aspect-1/1', 'aspect-2/1',
-  'bg-[url(./20s-scientists.jpg)]',
-  'bg-[url(./50s-scientists.jpg)]',
-  'bg-[url(./50s-scientists_240.jpg)]',
-  'bg-[url(./office-warping.png)]',
-  'bg-[url(./repeatable.png)]',
+  'bg-[url(../../images/20s-scientists.jpg)]',
+  'bg-[url(../../images/50s-scientists.jpg)]',
+  'bg-[url(../../images/50s-scientists_240.jpg)]',
+  'bg-[url(../../images/office-warping.png)]',
+  'bg-[url(../../images/repeatable.png)]',
```

Comment:
It appears that I UnoCSS can only allow our arbitrary class to include a relative path to the assets. So this is not working: `bg-[url(images/blah.jpg)]` and I needed to set the path like so: `bg-[url(../../images/blah.jpg)]`